recipe/meta.yaml

```diff
@@ -78,6 +79,7 @@ requirements:
     - libcusparse-dev
     {% endif %}
   run:
+    - ninja
```

Comment:
Ninja should not be added as a runtime dependency, xref https://github.com/conda-forge/deepspeed-feedstock/issues/1

```suggestion
```